packages/apollo-server-core/src/plugin/usageReporting/plugin.ts

```diff
@@ -598,6 +599,14 @@ export function ApolloServerPluginUsageReporting<TContext>(
               // End early if we aren't going to send the trace so we continue to
               // run the tree builder.
               didEnd(requestContext);
+            } else {
+              addTraceRequestStats({
```

Comment:
Can we consider having this be opt-in for the first release that runs it, so we and other users can try it out before it being on by default? (We can be explicit that the option is likely to default to true in a release or two.) It's just a lot of new code that could potentially have performance impacts, etc.

Reply:
I'm down for adding it as opt-in for the first release containing it, and noting in the changelog and option doc string that the option will become default true in a later release.